### PR TITLE
feature(common): added new feature (WEBSITES-15)

### PR DESCRIPTION
**Pull request by DanX3N0**
added feature (WEBSITES-15)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```
Comment (Ditmar) [inline .gitignore:21]: nice catch

File: src/App.styles.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { makeStyles } from '@material-ui/core';
+import mq from '../src/config/mq';
+
+const useStyles = makeStyles((theme) => ({
+   wrapper: {
+      backgroundColor: '#F1F0F0',
```
Comment (Ditmar) [inline src/App.styles.js]: please use the theme.palette, use the color from the palette

File: src/Theme/Theme.js [hunk 3]
```diff
@@ -26,13 +27,27 @@ const PoppinsBold = {
   fontWeight: 700,
   src: `url(${Poppins})`,
 };
+const AllertaStencilRegular = {
+  fontFamily: "AllertaStencil-Regular",
```
Comment (Ditmar) [inline src/Theme/Theme.js]: use simple quotes

File: src/Theme/Theme.js [hunk 4]
```diff
@@ -26,13 +27,27 @@ const PoppinsBold = {
...
+  fontStyle: "normal",
```
Comment (Ditmar) [inline src/Theme/Theme.js]: use simple quotes

File: src/Theme/Theme.js [hunk 5]
```diff
@@ -26,13 +27,27 @@ const PoppinsBold = {
...
+  fontWeight: 100,
+  src: `url(${AllertaStencil})`,
+};
+const AllertaStencilBold = {
+  fontFamily: "AllertaStencil-Bold",
```
Comment (Ditmar) [inline src/Theme/Theme.js]: use simple quotes

File: src/ui/components/listClients/AddAndSearch.jsx [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+import Avatar from './img/avatar.png';
+
+const AddAndSearch = () => {
+   const classes = useStyles();
+   return (
+      <>
+         <div className={classes.wrapper}>
+            <p className={classes.logo}>Frasier</p>
+            <div className={classes.searchForm}>
+               <div className={classes.lupa}>
```
Comment (Ditmar) [inline src/ui/components/listClients/AddAndSearch.jsx]: please use only ingles to call your vars you can use classes.magnifier

File: src/ui/components/listClients/AddAndSearch.jsx [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+                  <img src={Lupa} alt='lupa' />
+               </div>
+               <div className={classes.textAreaWidth}>
+                  <textarea className={classes.textArea} placeholder='Buscar'>
+                  </textarea>
+               </div>
+            </div>
+            <div>
+               <button className={classes.addButton}>
+                  <img className={classes.imgAdd} src={AddClient} alt='add-client' />
```
Comment (Ditmar) [inline src/ui/components/listClients/AddAndSearch.jsx]: rename this var to imageAdd

File: src/ui/components/listClients/AddAndSearch.jsx [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+               </div>
+               <div className={classes.userText}>
+                  <p className={classes.userName}>Jhon Jacob<br />
+                     jhon.jacob@gmail.com</p>
+               </div>
+            </div>
+         </div>
+         <div className={classes.hidden}>
+            <div className={classes.searchForm2}>
+               <div className={classes.lupa2}>
```
Comment (Ditmar) [inline src/ui/components/listClients/AddAndSearch.jsx]: The same here change the name of this variable to secondMagnifier

File: src/ui/components/listClients/AddAndSearch.jsx [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+                  <img src={Lupa} alt='lupa' />
+               </div>
+               <div className={classes.textAreaWidth2}>
+                  <textarea className={classes.textArea2} placeholder='Buscar'>
+                  </textarea>
+               </div>
+            </div>
+            <div className={classes.buttonHidden}>
+               <button className={classes.addButton2}>
```
Comment (Ditmar) [inline src/ui/components/listClients/AddAndSearch.jsx]: the same here addButton2 isn't a good name try to find another

File: src/ui/components/listClients/AddAndSearch.jsx [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+                  <img className={classes.imgAdd2} src={AddClient} alt='add-client' />
```
Comment (Ditmar) [inline src/ui/components/listClients/AddAndSearch.jsx]: the same here imgAdd2 isn't a good name try to find another

File: src/ui/components/listClients/AddAndSearch.styles.js [hunk 11]
```diff
@@ -0,0 +1,167 @@
...
+      },
+      [mq('md')]: {
+         display: 'flex',
+      },
+      color: '#FFFFFF',
+      width: '198px',
+      height: '46px',
+      display: 'flex',
+      flexDirection: 'row',
+      backgroundColor: '#000000',
```
Comment (Ditmar) [inline src/ui/components/listClients/AddAndSearch.styles.js]: if you don't find this color on the palette you can create a new one

File: src/ui/components/listClients/AddAndSearch.styles.js [hunk 12]
```diff
@@ -0,0 +1,167 @@
...
+         flexDirection: 'row',
+         paddingBottom: '30px',
+         justifyContent: 'center',
+      },
+      [mq('md')]: {
+         display: 'none',
+      },
+      backgroundColor: theme.palette.primary.main,
+   },
+   lupa2:{
```
Comment (Ditmar) [inline src/ui/components/listClients/AddAndSearch.styles.js]: change the name of this classes based in the comments above

File: src/ui/components/listClients/AddAndSearch.styles.js [hunk 13]
```diff
@@ -0,0 +1,167 @@
...
+      width: '10%',
+   },
+   searchForm2:{
```
Comment (Ditmar) [inline src/ui/components/listClients/AddAndSearch.styles.js]: change the name of this classes based in the comments above

File: src/ui/components/listClients/AddAndSearch.styles.js [hunk 14]
```diff
@@ -0,0 +1,167 @@
...
+      display: 'flex',
+      flexDirection: 'row',
+      width: 'calc(262 + 80% - 200px)',
+      height: ' 45px',
+      border: '1px solid #BFBFBF',
+      borderRadius: '22px',
+      marginLeft: '2%',
+      marginRight: '10%',
+   },
+   textAreaWidth2:{
```
Comment (Ditmar) [inline src/ui/components/listClients/AddAndSearch.styles.js]: change the name of this classes based in the comments above

File: src/ui/components/listClients/AddAndSearch.styles.js [hunk 15]
```diff
@@ -0,0 +1,167 @@
...
+      width: '70%',
+      height: '21px',
+   },
+   textArea2:{
```
Comment (Ditmar) [inline src/ui/components/listClients/AddAndSearch.styles.js]: change the name of this classes based in the comments above

File: src/ui/components/listClients/AddAndSearch.styles.js [hunk 16]
```diff
@@ -0,0 +1,167 @@
...
+      border: 'none',
+      width:'100%',
+      height: '21px',
+      outline: 'none',
+      resize: 'none',
+      '&::placeholder':{
+         color: '#BFBFBF',
+      },
+   },
+   addButton2:{
```
Comment (Ditmar) [inline src/ui/components/listClients/AddAndSearch.styles.js]: change the name of this classes based in the comments above

File: src/ui/components/listClients/AddAndSearch.styles.js [hunk 17]
```diff
@@ -0,0 +1,167 @@
...
+      color: '#FFFFFF',
+      width: '100%',
+      height: '46px',
+      display: 'flex',
+      flexDirection: 'row',
+      backgroundColor: '#000000',
```
Comment (Ditmar) [inline src/ui/components/listClients/AddAndSearch.styles.js]: use the palette color

File: src/ui/components/listClients/FilterBar.jsx [hunk 18]
```diff
@@ -0,0 +1,55 @@
+import useStyles from './FilterBar.styles';
+import ChecksComponent from './ChecksComponent';
+import PriceComponent from './PriceComponent';
+import FilterIcon from './img/filter-icon.png';
+import Material from './Material';
+
+const FilterBar = () => {
+   const data = {
```
Comment (Ditmar) [inline src/ui/components/listClients/FilterBar.jsx]: put this data in a mock file you can create a new folder   inside de src, called mockedData, please set the data there

**Review by Ditmar — CHANGES_REQUESTED** [review]
Really nice work Dani, but is a really minimun fix, good job

**Comment by olmedo90** [comment]
image

no es acorde al figma corrija el color del titulo

**Comment by olmedo90** [comment]
image

It is not according to Figma. correct the following: (color of the selected checkboxes, bold text of the selected checkbox, color of the price range bar)

**Comment by olmedo90** [comment]
image

It is according to Figma. continue with the same interest.👍

**Review by Ditmar — APPROVED** [review]
approved

**Pull request merged**